ddtrace/profiling/exporter/http.py

```diff
@@ -54,14 +55,22 @@ def __init__(self, exceptions):
         )
 
 
+def _get_api_key():
+    legacy = _attr.from_env("DD_PROFILING_API_KEY", "", str)()
```

Comment:
Just curious as to why we prioritize the old environment variable first?

Reply:
Not because it's older, but because it's more precise.
e.g. we do the same for any settings: if it's more precise for an integration than the general one, we pick the precise one, not the general setting.
I just applied the same here. :)